src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -52,14 +52,11 @@ protected String getQuery(SlingHttpServletRequest request) {
         // is not returning any results
         StringBuilder queryBuilder = new StringBuilder()
                 .append("select m.* from [nt:base] as m ")
-                    .append("INNER JOIN [pant:moduleVersion] as rev ON ISDESCENDANTNODE(rev, m) ")
+                    .append("INNER JOIN [pant:moduleLocale] as rev ON ISDESCENDANTNODE(rev, m) ")
                 .append("where (isdescendantnode(m, '/content/repositories') ")
                     .append("or isdescendantnode(m, '/content/modules') ")
                     .append("or isdescendantnode(m, '/content/sandbox')) ")
-                .append("AND m.[jcr:primaryType] = 'pant:module' ")
-                // look in ALL versions (all locales)
-                .append("AND (rev.[metadata/jcr:title] like '%" + searchParam + "%' ")
```

Comment:
This removes filtering by title and description.

Reply:
To draw clearer attention to it, it also makes `searchParam` a completely unused variable.

This would effectively remove the capability to search from our search page, turning it instead into a simple module listing page.